src/clj_kondo/impl/analyzer.clj

```diff
@@ -1532,6 +1532,11 @@
                         (test/analyze-cljs-test-async ctx expr)
                         ([clojure.test are] [cljs.test are] #_[clojure.template do-template])
                         (test/analyze-are ctx expr)
+                        ([clojure.spec.alpha def] [cljs.spec.alpha def])
```

Comment:
@borkdude Is this the right way to build in the s/def support? Or should I define a default  `:hooks` config for these somehow?  (I couldn't find anything else doing that, so I went this path).

Reply:
Yes, this is how built-in support for core macros is done.